envision/client.py

```diff
@@ -73,9 +75,18 @@ def __init__(
         num_retries: int = 5,
         wait_between_retries: float = 0.5,
         output_dir: str = None,
+        client_id: str = None,
     ):
         self._log = logging.getLogger(self.__class__.__name__)
-        client_id = str(uuid.uuid4())[:8]
+
+        current_time = datetime.now().strftime("%Y%m%d%H%M%S%f")[:-4]
```

Comment:
The readability of a timestamp like `2020122110132406` is really bad. Perhaps just `%H:%M:%S` is enough.